docs/conf.py

```diff
@@ -77,11 +78,13 @@ def __getattr__(cls, name):
 
 html_theme_options = {
     "announcement": """
-        <a style=\"text-decoration: none; color: white;\" 
+        <a style=\"text-decoration: none; color: white;\"
            href=\"https://opencollective.com/urllib3\">
-           <img src=\"_static/favicon.png\"/> Sponsor urllib3 v2.0 on Open Collective
+           <img src=\"{docs_root}/{image_path}\"/> Sponsor urllib3 v2.0 on Open Collective
```

Comment:
I realise approach assumes the docs are built and deployed on the same filesystem....which may not necessarily be the case.